pandas/core/frame.py

```diff
@@ -2837,6 +2838,75 @@ def to_parquet(
             **kwargs,
         )
 
+    def to_orc(
+        self,
+        path: FilePath | WriteBuffer[bytes] | None = None,
+        engine: Literal["pyarrow"] = "pyarrow",
```

Comment:
Do you expect that there will be multiple pyarrow-like modules? If not, you can probably remove this argument.

Reply:
Yes. There is also `pyorc` which can also read and write ORC files. I can add it once this PR is merged.